yangvalidator/v1/views.py

```diff
@@ -27,7 +27,7 @@
 from html import escape
 from io import StringIO
 from subprocess import CalledProcessError, call, check_output
-from tempfile import *
+from tempfile import *  # noqa: F403
```

Comment:
just `mkdtemp` seems to be used from `tempfile` so we can change this import to:
`from tempfile import mkdtemp`